### PR TITLE
View offer

### PR DESCRIPTION
**Pull request by Ollesun**

### CODE REVIEW

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/ViewOffer.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        if (menuItem.getItemId() == android.R.id.home) {
+            finish();
+        }
+        return super.onOptionsItemSelected(menuItem);
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.view_offer);
```
Comment (GeneralMustard) [inline goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/ViewOffer.java]: Change places on these two and I think you could extend ViewTemplate!

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/ViewOffer.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+import android.view.MenuItem;
+import android.widget.TextView;
+
+import com.goodpeople.gooddeeds.Model.Entities.IDeed;
+import com.goodpeople.gooddeeds.R;
+
+
+public class ViewOffer extends AppCompatActivity {
+    DeedController deedController = new DeedController();
```
[the remark this answers is not in the document]
Comment (Ollesun) [reply goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/ViewOffer.java]: Removed!

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/Entities/Deed.java [hunk 3]
```diff
@@ -10,6 +24,13 @@
     private Deed() {
     }
 
+    private Deed(Parcel in) {
+        givingAccount = in.readParcelable(Account.class.getClassLoader());
+        receivingAccount = in.readParcelable(Account.class.getClassLoader());
```
[the remark this answers is not in the document]
Comment (Ollesun) [reply goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/Entities/Deed.java]: As of right now, ViewOffer doesn't actually do anything with the attribute however so we can't really know for sure yet but I switched and It seems to work like usual!

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/ViewOffer.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+
+import android.widget.TextView;
+
+import com.goodpeople.gooddeeds.Model.Entities.IDeed;
+import com.goodpeople.gooddeeds.R;
+
+
+public class ViewOffer extends ViewTemplate {
+    private IDeed deed;
```
Comment (richardmarklund) [inline goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/ViewOffer.java]: You don't need to have this here. Maybe just use it as a local variable in the onCreate method to loosen the coupling?

**Pull request merged**